### PR TITLE
Bug 1608185 - wait for protocol initialization

### PR DESCRIPTION
**Pull request by djmitche**
Bugzilla Bug: [1608185](https://bugzilla.mozilla.org/show_bug.cgi?id=1608185)

### CODE REVIEW

File: workers/generic-worker/usage.go [hunk 1]
```diff
@@ -32,15 +32,19 @@ and reports back results to the queue.
...
     generic-worker show-payload-schema
     generic-worker new-ed25519-keypair      --file ED25519-PRIVATE-KEY-FILE` + customTargetsSummary() + `
     generic-worker --help
     generic-worker --version
 
   Targets:
-    run                                     Runs the generic-worker.
+    run                                     Runs the generic-worker.  Pass --with-worker-runner if
+											running under that service, otherwise generic-worker will
+											not communicate with worker-runner.
```
Comment (petemoore) [inline workers/generic-worker/usage.go]: Something gone awry with the formatting here.

Comment (djmitche) [reply workers/generic-worker/usage.go]: Hm, it's just tabs vs. spaces.  I thought Go preferred tabs?

**Comment by djmitche** [comment]
The test failures look like modifications to `go.mod`.  What would cause those?

**Comment by djmitche** [comment]
Ah, got it..

File: workers/generic-worker/usage.go [hunk 2]
```diff
@@ -32,15 +32,19 @@ and reports back results to the queue.
 
   Usage:
     generic-worker run                      [--config         CONFIG-FILE]
+                                            [--with-worker-runner]
                                             [--worker-runner-protocol-pipe PIPE]
                                             [--configure-for-aws | --configure-for-gcp | --configure-for-azure]` + installServiceSummary() + `
+
```
Comment (petemoore) [inline workers/generic-worker/usage.go]: I think this blank line broke docopt which indirectly caused the Windows failures. The broken tests on windows are calling a custom windows target which is defined in `customTargetsSummary()` on line 38. docopt doesn't see those options due to the blank line here, so returns the usage text.

**Comment by djmitche** [comment]
alles grün!

**Pull request merged**